Makefile

```diff
@@ -2,8 +2,14 @@
 MKDOCS_MATERIAL_VERSION := 9.1.4
 PORT ?= 8000
 
+ifeq ($(shell command -v podman 2> /dev/null),)
+    CONTAINER=docker
```

Comment:
can we call it runtime or engine or so ... container is definetly the wrong term here.

Reply:
Addressed in [320e60e](https://github.com/sdcio/docs/pull/67/commits/320e60ec4db851148011b0c762ae911fb3114cd2)